packages/jest-util/src/install_common_globals.js

```diff
@@ -12,45 +12,55 @@ import type {Global} from 'types/Global';
 
 function deepCopy(obj) {
   const newObj = {};
-  let value;
+
   for (const key in obj) {
-    value = obj[key];
-    if (typeof value === 'object' && value !== null) {
-      value = deepCopy(value);
+    if (obj.hasOwnProperty(key)) {
+      let value = obj[key];
+
+      if (typeof value === 'object' && value !== null) {
+        value = deepCopy(value);
+      }
+
+      newObj[key] = value;
     }
-    newObj[key] = value;
   }
+
   return newObj;
 }
 
-export default (global: Global, globals: ConfigGlobals) => {
-  // Forward some APIs
-  global.Buffer = Buffer;
-
-  // `global.process` is mutated by FakeTimers. Make a copy of the
-  // object for the jsdom environment to prevent memory leaks.
-  // Overwrite toString to make it look like the real process object
-  let toStringOverwrite;
-  if (Symbol && Symbol.toStringTag) {
-    // $FlowFixMe
-    toStringOverwrite = {
-      [Symbol.toStringTag]: 'process',
-    };
+export default (globalObject: Global, globals: ConfigGlobals) => {
+  let prototype = Object.getPrototypeOf(process);
```

Comment:
Based on https://github.com/nodejs/node/blob/master/lib/process.js I'm assuming there is no other way to get the prototype.